site/content/docs/5.0/components/offcanvas.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: docs
+title: Offcanvas
+description: Build hidden sidebars into your project for navigation, shopping carts, and more with a few classes and our JavaScript plugin.
+group: components
+toc: true
+---
+
+## How it works
+
+The offcanvas JavaScript plugin shows and hides sidebar on the left, right, or bottom of your viewport. Buttons or anchors are used as triggers that are attached to specific elements you toggle.
+
+Given how CSS handles animations, you cannot use `margin` or `translate` on a `.offcanvas` element. Instead, use the class as an independent wrapping element.
+
+## Example
+
+Click the buttons below to show and hide an offcanvas element via class changes:
+
+- `.offcanvas` hides content
+- `.offcanvas.show` shows content
+
+You can use a link with the `href` attribute, or a button with the `data-bs-target` attribute. In both cases, the `data-bs-toggle="offcanvas"` is required.
+
+{{< example >}}
```

Comment:
for examples, which are extremely long and complex-looking mainly due to the amount of example content in the off-canvas, consider perhaps not using `{{< example >}}` and instead doing a split of actual code vs shown code, eliding the content of the `offcanvas-body` as it's not really that interesting from a sample code perspective.

see how https://github.com/twbs/bootstrap/blob/main/site/content/docs/5.0/components/modal.md for instance does it for vertically centered modals (https://getbootstrap.com/docs/5.0/components/modal/#vertically-centered)